src/index.html

```diff
@@ -12,137 +12,285 @@
 ---
 
 <!-- ABOUT -->
-<section class="light-background">
-  <div class="main-container main-container--align-center layout-grid">
-    <div class="flex-container">
-      <h2>Meet BeakBox</h2>
-      <p>
-        Did you know that your parrot needs more than just basic toys to stay mentally sharp and happy? That's why we created BeakBox, a one-of-a-kind music toy that challenges your parrot's
-        cogniitive abilities and keeps them engaged all day long.
-      </p>
-
-      <div class="flex">
-        <a class="button" href="/how-it-works/" aria-label="Learn more about how BeakBox works">How it works</a>
+<section>
+  <div class="main-container main-container--align-center main-container--narrow">
+    <div class="flex-container flex-container--center-content">
+      <h2>Your parrots need <span style="color: #ff662b">more than basic toys</span></h2>
+      <p>That's why we created BeakBox, a one-of-a-kind music toy that challenges your parrot's cognitive abilities and
+        keeps them engaged all day long.</p>
+      <div>
+        <iframe width="560" height="315" loading="lazy" src="https://www.youtube.com/embed/_xL-Lp1K0F4"
+          title="YouTube video introducing the BeakBox" frameborder="0"
+          allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share" allowfullscreen></iframe>
       </div>
     </div>
+</section>
+
+<section>
+  <div class="bird-side main-container main-container--narrow banner">
+    <div class="flex-container flex-container--center-content">
+      <h2>BeakBox</h2>
+      <h3>Bird side</h3>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="human-side main-container main-container--narrow banner">
+    <div class="flex-container flex-container--center-content">
+      <h2>BeakBox</h2>
+      <h3>Human side</h3>
+    </div>
+  </div>
+</section>
 
-    <div>
-      <iframe
-        width="560"
-        height="315"
-        loading="lazy"
-        src="https://www.youtube.com/embed/_xL-Lp1K0F4"
-        title="YouTube video introducing the BeakBox"
-        frameborder="0"
-        allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"
-        allowfullscreen
-      ></iframe>
+<section class="main-container main-container--narrow main-container--align-center">
+  <div class="flex-container flex-container--center-content">
+    <h2>How BeakBox <span style="color: #ff662b">works</span></h2>
```

Comment:
I think this is in a number of places - probably best to have a CSS rule for it.